justfile

```diff
@@ -52,3 +110,50 @@ clear-gh-caches:
             | for ID in `jq '.actions_caches[].id'`; \
               do echo "Deleting $ID"; \
                  gh api --method DELETE /repos/tlspuffin/tlspuffin/actions/caches/$ID | echo; done
+
+# RECIPE: `act`
+#
+# NOTE: additional arguments are passed to the native `act` command
+# NOTE: set ACT_EVENT_NAME/ACT_EVENT_FILE to emulate custom trigger events
+#
+# Examples:
+# - list workflow triggered by the command and stops:
+#   $ just act --list
+#
+# - do a dry-run:
+#   $ just act -n
+#
+# - run only the configure job:
+#   $ just act -j configure
+#
+# - emulate a pull_request against the main branch:
+#   $ just ACT_EVENT_NAME=pull_request ACT_EVENT_FILE=${PWD}/.github/act/events/pr_to_main.json act
```

Comment:
I think it would be great to have a reference tot his in the README

Reply:
Yes, good point. I have taken note to work on this when I do the PR related to tlspuffin's documentation.